[32mINFO    [0m root:conftest.py:12 开始计算
[32mINFO    [0m root:test_calculator.py:27 异常情况加法运算：0+s相加 结果TypeError异常
[32mINFO    [0m root:conftest.py:15 结束计算